lci_strategic_plugin/include/lci_strategic_plugin/lci_strategic_plugin_config.h

```diff
@@ -65,6 +65,21 @@ struct LCIStrategicPluginConfig
   //! The minimum period in seconds which a maneuver plan must cover if the plugin wishes to control the whole plan
   double min_maneuver_planning_period = 15.1;
 
+  //! Double: Approximate update time interval of carma streets
+  double delta_t = 1.0;
+
+  //! Double: Vehicle reaction time to a received schedule in seconds (approximate value, only used for communication with the schedule)
+  double reaction_time = 5.0;
```

Comment:
This is a pretty high default value. I think 2 would be more appropriate. Also change in yaml